oci/layout/oci_dest.go

```diff
@@ -27,11 +27,21 @@ func newImageDestination(ref ociReference) (types.ImageDestination, error) {
 	if ref.image == "" {
 		return nil, errors.Errorf("cannot save image with empty image.ref.name")
 	}
+
 	index := imgspecv1.Index{
 		Versioned: imgspec.Versioned{
 			SchemaVersion: 2,
 		},
 	}
+
+	if indexExists(ref.dir) {
```

Comment:
This would be cleaner as `if indexExists() { index = … } else { index = … }`, rather than always setting `index` only to overwrite it.

Reply:
fair enough